fili-core/src/test/groovy/com/yahoo/bard/webservice/data/PartialDataHandlerSpec.groovy

```diff
@@ -69,73 +57,47 @@ class PartialDataHandlerSpec extends Specification {
          * dim1 is missing four days of data internally, dim2 and dim3 are complete over the period and page_views is
          * starts inside the dim1 hole and goes to the end of the period.
          */
-        SegmentMetadata segments = new SegmentMetadata(
-                [("userDeviceType"): buildIntervals(["2014-07-01/2014-07-09","2014-07-11/2014-07-29"]) as LinkedHashSet,
-                 ("property"): buildIntervals(["2014-07-01/2014-07-29"]) as LinkedHashSet,
-                 ("os"): buildIntervals(["2014-07-01/2014-07-29"]) as LinkedHashSet],
-                [("page_views"): buildIntervals(["2014-07-04/2014-07-29"]) as Set]
+        Map<Column, Set<Interval>> segmentIntervals = [
+                (new Column("userDeviceType")): buildIntervals(["2014-07-01/2014-07-09","2014-07-11/2014-07-29"]).toSet(),
```

Comment:
It's more idiomatic to use the `as` operator, rather than the `toSet()`method:

```groovy
 (new Column("userDeviceType")): buildIntervals(["2014-07-01/2014-07-09","2014-07-11/2014-07-29"]) as Set
```